runtime/near-vm-runner/src/imports.rs

```diff
@@ -264,4 +277,7 @@ wrapped_imports! {
     // ###############
     validator_stake<[account_id_len: u64, account_id_ptr: u64, stake_ptr: u64] -> []>,
     validator_total_stake<[stake_ptr: u64] -> []>,
+    // ###############
+    // An example to add a protocol feature guarded host method
+    // #["protocol_feature_evm", EVM] test_api<[a: u64] -> []>,
```

Comment:
This example seems to be confusing, as evm is not executed in near-vm-runner. Should we use another information here?
Also, consider add a test to guarded api?

Reply:
Yes. The test is here: https://github.com/near/nearcore/pull/3827

But I need to compile a contract with this import and check that it fails to run in the previous protocol version and runs in the new protocol version.

We can do this for a real new import. For example `burn_near`